include/openthread/instance.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @note This number versions both OpenThread platform and user APIs.
  *
  */
-#define OPENTHREAD_API_VERSION (235)
+#define OPENTHREAD_API_VERSION (236)
```

Comment:
@samkumar , can you rebase and update this value again?

Reply:
Thanks @jwhui for taking look. I just rebased the code and updated this pull request.